server/router/test.js

```diff
@@ -28,13 +28,16 @@ describe('GET /conference', () => {
   });
 });
 
-describe('GET /community', () => {
-  it('/renders OK', () => {
-    useDummyData();
-    return request(app)
-      .get('/community')
-      .expect(200);
-  });
-});
+// describe.only('GET /community', () => {
```

Comment:
This should be marked as pending. Never comment out tests or we don't know it's pending.